rpa-mapper/src/test/resources/admissions/individual_full_admission_by_instalments_rpa_case.json

```diff
@@ -0,0 +1,34 @@
+{
+  "caseNumber": "000CM001",
+  "responseSubmittedOn": "26-Apr-2018",
+  "defenceResponse": "FULL_ADMISSION",
+  "amountAdmitted": null,
+  "defendant": {
+    "type": "Individual",
+    "name": "John Rambo",
+    "businessName": null,
+    "contactPerson": null,
+    "addressAmended": false,
+    "address": {
+      "line1": "52",
+      "line2": "Down Street",
+      "line3": "Salford",
+      "city": "Manchester",
+      "postcode": "DF1 3LJ"
+    },
+    "emailAddress": "j.smith@example.com",
+    "correspondenceAddress": null,
+    "dateOfBirth": "02-Jan-1968",
```

Comment:
Don't need to be ISO8601 formatted?

Reply:
This is the formatting for dates we've been using. Not too sure what you meant.